build.xml

```diff
@@ -49,30 +50,45 @@
                 
     <!-- test-compile -->
     <target name="test-compile" depends="resolve-classpath">
-        <javac includeantruntime="false" srcdir="${test.dir}" destdir="${class.dir}"
+        <javac includeantruntime="false" srcdir="${test.dir}" destdir="${class.dir}/test"
             debug="true" debuglevel="lines,vars,source" fork="true">
             <compilerarg line="-J-Xmx4g" />
             <classpath refid="lib.classpath" />
             <classpath>
+                <pathelement location="${class.dir}" />
                 <pathelement location="${testcfg.dir}" />
             </classpath>
         </javac>
-        <copy todir="${class.dir}">
+        <copy todir="${class.dir}/test">
             <fileset dir="${test.dir}">
                 <include name="**/*.txt" />
                 <include name="**/*.tokens" />
                 <include name="**/*.list" />
                 <include name="**/*.xls*" />
+                <include name="**/*.ods" />
```

Comment:
hmm how did ods tests run?

Reply:
In your earlier version, all the tests were running using the files in `test/resources/`, even though you were copying them things to `classes/` already.  When I decided to create the Windows files automatically, it was obvious they belonged in one of `build.xml`'s output directories, not it's one of its inputs.  The `.ods` files weren't being copied, but your code didn't care, because it was ignoring the test stuff in `classes/`.